core/client/src/client.rs

```diff
@@ -691,6 +664,54 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 		Ok(ImportResult::Queued)
 	}
 
+	fn block_execution(
+		&self,
+		import_headers: &PrePostHeader<Block::Header>,
+		origin: BlockOrigin,
+		hash: Block::Hash,
+		body: Option<Vec<Block::Extrinsic>>,
+		transaction: &B::BlockImportOperation,
+	) -> error::Result<(Option<StorageUpdate<B, Block>>, Option<Option<ChangesUpdate>>, Option<Vec<(Vec<u8>, Option<Vec<u8>>)>>)> where
```

Comment:
style: could split the tuple args each onto their own line with another level of indentation

Reply:
resolved